api/post_test.go

```diff
@@ -615,7 +613,7 @@ func TestSearchPostsFromUser(t *testing.T) {
 		t.Fatalf("wrong number of posts returned %v", len(result.Order))
 	}
 
-	if result := Client.Must(Client.SearchPosts("from: " + user2.Username + " in:" + channel1.Name)).Data.(*model.PostList); len(result.Order) != 1 {
+	if result := Client.Must(Client.SearchPosts("from: " + user2.Username + " in:" + channel1.Name)).Data.(*model.PostList); len(result.Order) != 0 {
```

Comment:
It's just those two cases that return 0 that concern me since they're the only ones that test using both search flags at once. The other ones are fine as is, but I thought it might be easier to just revert the changes to the expected numbers and add some extra posts programmatically.